.changeset/tough-dancers-do.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/address': major
```

Comment:
This is not a breaking change so it does not need to be a major version number. This should be a `minor` release.

Reply:
@BPScott 
Thank you. I have made the changes.
https://github.com/Shopify/quilt/pull/2662/commits/5da45024046325ffd8e71940f5e1f7b2aa3ec0cb